packages/calling-stateful-client/src/Converter.ts

```diff
@@ -191,9 +194,13 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
       currentSpokenLanguage: '',
       isCaptionsFeatureActive: false,
       startCaptionsInProgress: false,
-
       captionsKind: _isTeamsCall(call) ? 'TeamsCaptions' : 'Captions'
     },
+    /* @conditional-compile-remove(rtt) */
+    realTimeTextFeature: {
+      realTimeTexts: {},
```

Comment:
What is this object supposed to be? I can see it's an object. 
Is this supposed to be a list of realTimeTextMessages? Then this should be renamed as realTimeTextMessages

Reply:
It's split into messages inside the object 
realTimeTexts: {
    completedMessages?: RealTimeTextInformation[];
    currentInProgress?: RealTimeTextInformation[];
    myInProgress?: RealTimeTextInformation;
  };